test/modules/op/depthwise_conv2d.py

```diff
@@ -29,6 +32,28 @@ def get_example_inputs(self):
         return (torch.randn(1, 8, 64, 64),)
 
 
+@tag.use_onert
+class SimpleDepthwiseConvDynamicShape(torch.nn.Module):
```

Comment:
@jinevening Thanks for the comment! I added the logic to add_tensor_from_scratch.

<img width="796" height="1094" alt="image" src="https://github.com/user-attachments/assets/a35118f1-c05b-4282-aadb-456c2953c93b" />

This test ensures that any intermediate tensor made by `add_tensor_from_scratch` has a dynamic shape - padding in this case.